packages/flutter/lib/src/widgets/focus.dart

```diff
@@ -79,8 +96,15 @@ class Focus extends StatefulComponent {
 
   final Widget child;
 
+  /// The key that currently has focus globally in the entire focus tree.
+  ///
+  /// This field is always null when asserts are disabled.
```

Comment:
Yeah, I don't really know that part of the story.  Maybe I'll convince you to fill that part in.  :)